blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala

```diff
@@ -22,6 +22,31 @@ import scala.collection.immutable
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
 
+/**
+ * BlazeBuilder is the component for the builder pattern aggregating
+ * different components to finally serve requests.
+ *
+ * Variables: 
```

Comment:
I think if you put a `@param` annotation on these, it'll do the right thing for scaladoc.